bin/dev_setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+echo "Checking java 1.8.x"
```

Comment:
this seems a little too specific, since presumably we'd want to check for java 1.8 or higher, right?  i'd almost rather leave the version check off than have to update it to include a new version or something every time java had a new version release.